train_mnist.py

```diff
@@ -126,29 +126,15 @@ def scscn(x, num, num_conv):
             l = int(h % (m*n))
             j = int(l / n)
             k = int(l % n)
```

Comment:
These should be expressed in a clearer manner.

Reply:
That's a stupid thing, I will remove it soon.